### PR TITLE
Record blocknumber when anchoring

### PR DESCRIPTION
**Pull request by vimukthi-git**

### CODE REVIEW

File: test/Gas.test.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ contract("Gas costs", function (accounts) {
     describe("Check the gas cost for preCommit and commit with the identity proxy for ACTION key", async function () {
 
         const preCommitMaxGas = 95000;
-        const commitMaxGas = 80000;
+        const commitMaxGas = 85000;
```
Comment (vimukthi-git) [inline test/Gas.test.js]: @rdinicut

File: test/AnchorRepository.test.js [hunk 2]
```diff
@@ -117,7 +117,9 @@ contract("AnchorRepository", function (accounts) {
             assert.equal(bufferToHex(hashedAnchorId), web3.utils.toHex(response[0]));
             assert.equal(documentRoot, response[1]);
 
-
+            let blckNoResponse = await this.anchorRepository.getAnchoredBlockById.call(hashedAnchorId, callOptions);
+            assert.equal(bufferToHex(hashedAnchorId), web3.utils.toHex(blckNoResponse[0]));
+            assert.isAbove(0,  web3.utils.BN(blckNoResponse[1]));
```
Comment (vimukthi-git) [inline test/AnchorRepository.test.js]: @rdinicut whats the best way to verify this block number?

Comment (rdinicut) [reply test/AnchorRepository.test.js]: Why do we need bytes32 instead of uint for the block number?

**Comment by vimukthi-git** [comment]
merging with approval from @rdinicut @lucasvo ;)

**Pull request merged**